src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
```

Comment:
Consider adding a check to ensure that the `value` is a number before adding it to `sum`. This will prevent potential issues if non-numeric values are passed to the `adder` function.